keras/layers/core.py

```diff
@@ -90,10 +90,11 @@ class Dropout(Layer):
         - [Dropout: A Simple Way to Prevent Neural Networks from Overfitting](http://www.cs.toronto.edu/~rsalakhu/papers/srivastava14a.pdf)
     """
 
-    def __init__(self, p, noise_shape=None, seed=None, **kwargs):
+    def __init__(self, p, noise_shape=None, seed=None, active_in='train', **kwargs):
```

Comment:
Updated !

Dropout in inference phase is a [proposed] way to introduce stochasticity in a model.

For instance: Isola et. al [https://arxiv.org/pdf/1611.07004v1.pdf] use this technique in order to introduce some randomness in the generator model of a conditional Generative Adversarial Network. They, however, report that the effect at the output is minor.